src/main/webapp/resources/css/main.css

```diff
@@ -125,7 +125,10 @@ span.sub {
 .error-text {
     color: red;
 }
-#long-text {
+
+.long-text {
+    width: 100%;
+    display: inline-block;!important;
```

Comment:
Please check correctness of this expression. I think theme should be space instead of `;` before `important`.